### ISSUE TITLE
Orbot is not detected for connection over tor

### ISSUE THREAD

**Issue by kaikli** [open-keychain/open-keychain#2824]
It is not possible to enable the connection over tor because the existing Orbot installation is not detected.

## Expected Behavior
It should be possible to enable the tor option in the settings with a existing Orbot installation.

## Current Behavior
I only see the installation dialog for Orbot after trying to enable the tor option.

## Steps to Reproduce (for bugs)
1. Install Orbot from f-droid
2. Enable the tor option in the settings

## Context
It seems that Orbot is not detected properly.

## Your Environment

* Android Version: 13
* Device Model: Pixel 6a
* OpenKeychain Version: 5.8.2
* Orbot Version: 16.6.4-RC-1-tor.0.4.7.11
* From Google Play or F-Droid?: F-Droid (both)

**Comment by M-um-p-i-t-z** [comment]
You can use the HTTP Proxy as workround so far, I can confirm this with Orbot Version 17.0.0-RC-1

**Comment by lu-ckystars** [comment]
yes but it wont use the hidden service with the proxy workaround, and you can also use socks on 9050 of course.
